### PR TITLE
Update IRC links to point to Godot Contributors Chat

### PR DESCRIPTION
**Pull request by snailrhymer**
Replace links to IRC with links to Contributor's Chat in contributing docs for 3.3 (to match 3.4 and above).

Or would link to Libera.Chat be better to match #5547?

### CODE REVIEW

**Comment by mhilbrunner** [comment]
Hi! Calinou already cherrypicked https://github.com/godotengine/godot-docs/pull/5547 to 3.3, so this is superseded by that. :) Still, thanks for taking the time to contribute!

File: community/contributing/updating_the_class_reference.rst [hunk 1]
```diff
@@ -310,7 +310,7 @@ I don't know what this method does!
 
 No problem. Leave it behind, and list the methods you skipped when you request a pull of your changes. Another writer will take care of it.
 
-You can still have a look at the methods' implementation in Godot's source code on GitHub. Also, if you have doubts, feel free to ask on the `Q&A website <https://godotengine.org/qa/>`__ and on IRC (freenode, #godotengine).
+You can still have a look at the methods' implementation in Godot's source code on GitHub. Also, if you have doubts, feel free to ask on the `Q&A website <https://godotengine.org/qa/>`__ and on the `Contributors Chat <https://chat.godotengine.org/>`__.
```
Comment (Calinou) [inline community/contributing/updating_the_class_reference.rst:313]:
```suggestion
You can still have a look at the methods' implementation in Godot's source code on GitHub. Also, if you have doubts, feel free to ask on the `Q&A website <https://godotengine.org/qa/>`__ and on the `Godot Contributors Chat <https://chat.godotengine.org/>`__.
```

File: community/contributing/updating_the_class_reference.rst [hunk 2]
```diff
@@ -119,7 +119,7 @@ Another option is to delete the branch you're working on, synchronize the master
     git pull --rebase upstream master
     git checkout -b your-new-branch-name
 
-If you're feeling lost by now, come to our `IRC channels <https://webchat.freenode.net/?channels=#godotengine>`_ and ask for help. Experienced Git users will give you a hand.
+If you're feeling lost by now, come to our `Contributors Chat <https://chat.godotengine.org/>`_ and ask for help. Experienced Git users will give you a hand.
```
Comment (Calinou) [inline community/contributing/updating_the_class_reference.rst:122]:
```suggestion
If you're feeling lost by now, come to our `Godot Contributors Chat <https://chat.godotengine.org/>`_ and ask for help. Experienced Git users will give you a hand.
```

File: community/contributing/pr_workflow.rst [hunk 3]
```diff
@@ -364,7 +364,8 @@ On that line, there is a "Pull request" link. Clicking it will open a form
 that will let you issue a pull request on the ``godotengine/godot`` upstream
 repository. It should show you your two commits, and state "Able to merge".
 If not (e.g. it has way more commits, or says there are merge conflicts),
-don't create the PR, something went wrong. Go to IRC and ask for support :)
+don't create the PR, something went wrong. Go to the 
+`Contributors Chat <https://chat.godotengine.org/>`_ and ask for support :)
```
Comment (Calinou) [inline community/contributing/pr_workflow.rst:368]:
```suggestion
`Godot Contributors Chat <https://chat.godotengine.org/>`_ and ask for support :)
```

**Comment by snailrhymer** [comment]
Sorry, I don't think I've got any idea how to fix whatever's caused the check failure.

***

Should the change `Contributors Chat -> Godot Contributors Chat` also be made on the master branch?

**Comment by Calinou** [comment]
The CI failure is unrelated to this pull request. I fixed it in https://github.com/godotengine/godot-docs/commit/09a47f500ced5c67338a0e8c0aedb4bd1230d673 :slightly_smiling_face:

**Comment by mhilbrunner** [comment]
Thanks for this, and thanks Calinou for reviewing and fixes!

**Pull request merged**